src/main/scala/gitbucket/core/controller/api/ApiRepositoryContentsControllerBase.scala

```diff
@@ -13,10 +13,13 @@ import scala.util.Using
 trait ApiRepositoryContentsControllerBase extends ControllerBase {
   self: ReferrerAuthenticator with WritableUsersAuthenticator with RepositoryCommitFileService =>
 
-  /*
+  /**
    * i. Get the README
    * https://developer.github.com/v3/repos/contents/#get-the-readme
    */
+  get("/api/v3/repos/:owner/:repository/readme")(referrersOnly { repository =>
+    getContents(repository, "README.md", params.getOrElse("ref", repository.repository.defaultBranch))
```

Comment:
This implementation does not work if the repository has a "readme.md" or "README.MD", ...
GitHub allows case-insensitive README file.

Reply:
@SIkebe  Thank you for your review!
I add option to `getContents` for matching filename by low-case. https://github.com/gitbucket/gitbucket/pull/2535/commits/10d97a43f9760496ed1a73c580afbe2d4ee10c2b